new quickstart guides (#3060)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/useaccesstoken.md

> The recommended way to add your access token to HTTP calls in Angular is to use an interceptor. To follow security best practices, the access token should only be added on calls to allowed origins.

Reviewer: ```suggestion
The recommended way to add your access token to HTTP calls in Angular is to use an interceptor. To follow security best practices, you should only add the access token on calls to allowed origins.
```

```ts
import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
import { AuthInterceptor } from './auth.interceptor';
```

3. Register the interceptor in `app.module.ts` by adding the following into the `providers` array:

```ts
{ provide: HTTP_INTERCEPTORS, useClass: AuthInterceptor, multi: true }
```

4. Update `auth.interceptor.ts` to add the access code when calling a trusted origin:

```ts
import { Inject, Injectable } from '@angular/core';
import {
  HttpRequest,
  HttpHandler,
  HttpEvent,
  HttpInterceptor
} from '@angular/common/http';
import { Observable } from 'rxjs';
import { OKTA_AUTH } from '@okta/okta-angular';
import { OktaAuth } from '@okta/okta-auth-js';

@Injectable()
export class AuthInterceptor implements HttpInterceptor {

  constructor(@Inject(OKTA_AUTH) private _oktaAuth: OktaAuth) {}

  intercept(request: HttpRequest<unknown>, next: HttpHandler): Observable<HttpEvent<unknown>> {
    return next.handle(this.addAuthHeaderToAllowedOrigins(request));
  }

  private addAuthHeaderToAllowedOrigins(request: HttpRequest<unknown>): HttpRequest<unknown> {
    let req = request;
    const allowedOrigins = ['http://localhost'];

>     if(!!allowedOrigins.find(origin => request.url.includes(origin))) {

Reviewer 2: ```suggestion
    if (!!allowedOrigins.find(origin => request.url.includes(origin))) {
```

Author: changed
